_docs/troubleshooting.markdown

```diff
@@ -1,46 +1,46 @@
 ---
 title: Troubleshooting
+order: 4
 ---
 
-{% include doc-shortcut-tip.html %}
+{% include h title="Updating" %}
 
-{% include h title="Are you experiencing problems after update?" %}
+Normally, the plugin update should go seamlessly. In case of any issues,
+ try refreshing the project (click **Refresh** in the Cargo tool window) or rebuilding indices (
+**File | Invalidate caches / Restart** from the main menu).
+ 
+Previous versions of the plugin are availabe at the
+[JetBrains plugin repo](https://plugins.jetbrains.com/plugin/8182).
 
-In general, plugin updates should go smoothly. Though, if you experience some
-weird behavior, try reimporting your project or/and rebuilding indices (using
-**Invalidate caches/Restart** action). Previous versions are available at
-[https://plugins.jetbrains.com/plugin/8182](https://plugins.jetbrains.com/plugin/8182).
 
+{% include h title="Toolchain setup" %}
 
-{% include h title="Is Cargo project structure correct?" %}
+To make sure your toolchain is set up correctly, 
+go to `Settings / Preferences | Languages & Frameworks | Rust` and check the `Toolchain location` 
+and `Standard library` paths.
 
-At the moment, we support only Cargo based projects. Use Cargo toolbar to see
```

Comment:
Let's keep note about "we support only Cargo based projects". Although around 99% of Rust projects use Cargo, there are examples where another build system is used (for example, Bazel)